hnn_core/network.py

```diff
@@ -190,8 +195,17 @@ def __init__(self, params):
             'L5_basket',
             'L5_pyramidal',
         ]
-        self.feedname_list = []  # no feeds defined yet
-        self.feed_times = dict()
+
+        # external drives and biases
+        self.external_drives = dict()
+        self.external_biases = dict()
+        self.drive_event_times = dict()
+
+        # XXX are 'common' drives _always_ present in params?
+        p_common, p_unique = create_pext(self.params, self.params['tstop'])
+        # XXX needed for parconnect! redo cleaner (this PR)
+        self._p_common, self._p_unique = p_common, p_unique
+        self._n_common_feeds = 0
```

Comment:
This still needs some thought: the "common" and "unique" parameter sets contain the connectivity information needed in `Builder` by `parconnect`. I feel there's a better way of handling this, but it may be massively invasive on the whole codebase (so perhaps later PR?).

Reply:
I wonder if `self._n_common_feeds` attribute can go away? Why do we need it?